.github/workflows/docbuild-and-upload.yml

```diff
@@ -40,12 +40,19 @@ jobs:
     - name: Build Pandas
       uses: ./.github/actions/build_pandas
 
+    - name: Install build dependencies
+      run: |
+        python -m pip install -r web/pandas/requirements.txt
```

Comment:
I think it's probably fine to add those to our main `environment.yml`. We already have jupyter related dependencies there, and will make things a bit simpler. A comment to explain the two dependencies are used for the interactive shell (or a section in the file) would be useful.